### PR TITLE
RAC-5682 - Redfish Compliance, correct systems/ethernetinterface odata links for non-dell

### PR DESCRIPTION
**Pull request by larry-dean**
* Correct tests to allow ethernetinterfaces for non-dell devices

### CODE REVIEW

File: test/tests/redfish10/test_redfish10_api_systems.py [hunk 1]
```diff
@@ -149,10 +149,7 @@ def test_redfish_v1_systems_id_ethernetinterfaces(self):
         # Only works for Dell servers with microservices
```
Comment (jimturnquist) [inline test/tests/redfish10/test_redfish10_api_systems.py]: This comment should be updated, correct?  This test applies to all servers now.

Comment (larry-dean) [inline test/tests/redfish10/test_redfish10_api_systems.py]: Will fix

**Comment by larry-dean** [comment]
Changes pushed up

**Comment by jimturnquist** [comment]
test this please

**Comment by panpan0000** [comment]
:+0!

**Comment by anhou** [comment]
this PR is going to fix MasterCI #291 failure which would cause 'MergeFreezer' red.

**Pull request merged**